controllers/usersController.js

```diff
@@ -126,11 +125,9 @@ const updateSelf = async (req, res) => {
     }
 
     const user = await userQuery.addOrUpdate(req.body, userId)
-    const userWallet = await fetchWallet(userId)
-    if (!userWallet.id) {
-      await createWallet(userId, { dinero: 1000 })
-    }
-    if (!user.isNewUser) {
```

Comment:
Once a new user is signed up, there are additional tasks we may want to do
- Initialize their wallet
- Update the number of users
- Show it on members page, etc

Those post-signup events should exist in a separate function